agent.go

```diff
@@ -1355,6 +1355,8 @@ func newContainerCb(pod *pod, data []byte) error {
 				MemorySwappiness: nil,
 				AllowAllDevices:  nil,
 				AllowedDevices:   configs.DefaultAllowedDevices,
+				CpuQuota:         int64(payload.Constraints.CPUQuota),
```

Comment:
I guess it's unlikely to be an issue, but this _could_ overflow, so could be written as:

```
if payload.Constraints.CPUQuota > uint64(math.MaxInt64) {
    return fmt.Errorf("CPU quota too big: %d", payload.Constraints.CPUQuota)
}

cpuQuota := int64(payload.Constraints.CPUQuota)
```